tests/runfiles/common.run

```diff
@@ -713,6 +713,10 @@ tests = ['nopwrite_copies', 'nopwrite_mtime', 'nopwrite_negative',
     'nopwrite_varying_compression', 'nopwrite_volume']
 tags = ['functional', 'nopwrite']
 
+[tests/functional/nvlist]
+tests = ['packed']
+tags = ['functional', 'nvlist']
```

Comment:
Rather than creating a new test group I think it makes some sense to add this to the `functional/libzfs` group.  The `libzfs_input` test case here verifies the ZFS `ioctl()` interface remains unchanged, which is a similar sort of test.  Either way, it's great to see a test case for this!

Reply:
I could certainly do that, but it's part part of libzfs and instead is a nominally standalone library. I wonder if it should instead be in `tests/functional/libnvpair`.

Reply:
That's true, I was just hoping to avoid adding another top level test group.  That said, I'm fine with either `libnvpair` or `nvpair` as you've already done.